apiv2/api/openapi/openapi.yaml

```diff
@@ -4980,12 +4980,11 @@ components:
           $ref: '#/components/schemas/OperatingSystemResource'
         desiredOs:
           title: desired_os
-          description: OS resource that should be installed to this Instance.
-          readOnly: true
+          description: (OPTIONAL) Deprecated, will be removed in EMF v3.2.0, use OSUpdatePolicy for driving day2, and os for day0 operations instead. OS resource that should be installed to this Instance.
```

Comment:
is the (OPTIONAL) manually added? Have we done this consistenly?

Reply:
restored the previous behaviour